data/transport missions.txt

```diff
@@ -1867,6 +1872,297 @@ mission "Terraforming 5"
+	passengers 6
+	cargo "scientific equipment" 10
+	blocked "You need <capacity> in order to take on the next mission. Return here when you have the required space free."
+	to offer
+		has "Terraforming 8: done"
+	
+	on offer
+		conversation
+			`The spaceport is crowded with ships and people when you land. Amy's publication must have stirred up quite the commotion among terraforming scientists.`
+			`	Spaceport security clears a path to your ship for Amy, Nolan, and four others. You notice that a few among the crowd of people are jeering and holding signs, one reading "Don't asteroid my planet!" and a half-eaten sandwich gets thrown in her path as she approaches.`
+			`	"Not everyone is as happy with the prospect of affordable terraforming as we are," Amy says to you as she boards your ship along with suitcases full of scientific supplies. "But that's no reason to stop, now is it?"`
+			`	Amy introduces you to the four other people that she brought on board. Two of them are terraforming experts who helped Amy and Nolan with their research. The other two are representatives: one sent by the Republic Parliament in order to oversee the project, and another from Tundra.`
+			`	"Once you're ready, Captain, please bring us to <destination>. Tundra was a tropical world millions of years ago, but became cold after a cataclysmic event of some sort. There's a good chance that we may be able to nudge the planet back toward a warmer climate if we do this right."`
+					accept
+	
+	on complete
+		dialog 
+			`The journey to Tundra was spent by your passengers discussing the possible methods to terraform the planet. The Republic representative was none too pleased when Nolan, perhaps jokingly, suggested that we crash an asteroid into the planet.` 
+			`	Everyone except for the Tundra native begins to shiver after stepping out of your ship onto the planet, even while inside a protective dome. "Meet us in the spaceport bar in a few hours. We need to finalize our plans of how we're going to execute this experiment."`
+
+
+
+mission "Terraforming 10"
+	name "Terraforming Tundra"
+	description "Bring the Republic representative overseeing the terraforming project to <destination> where he will attempt to find an alternative solution to terraforming Tundra that does not involve an asteroid."
+	destination "Earth"
+	passengers 1
+	to offer
+		has "Terraforming 9: done"
+	
+	on offer
+		conversation
+			`Amy, Nolan, and the other two scientists tell you the known history of Tundra's climate. Millions of years ago, an asteroid impact or large volcanic eruption altered the planet's atmosphere. Now the planet no longer retains enough heat from its host star in order to stay warm.`
+			`	"Tundra's snow covered surface only exacerbates the problem, reflecting most of the star's light back into space." Nolan explains more. "This means that we need to both alter the atmosphere in order for it retain more heat and alter the surface in order for it to reflect less heat."
```

Comment:
So is this whole passage spoken by Nolan? If so, we should probably move the Nolan part to the front, or change that first period to a comma and remove `more` in `Nolan explains more.`